pkg/util/yaml/yaml.go

```diff
@@ -0,0 +1,39 @@
+package yaml
+
+import (
+	"bytes"
+	"fmt"
+
+	yamlpatch "github.com/krishicks/yaml-patch"
+
+	yaml "gopkg.in/yaml.v2"
+)
+
+func Decode(ba []byte) (yamlpatch.Container, error) {
+	decoder := yaml.NewDecoder(bytes.NewReader(ba))
+	var iface interface{}
+	err := decoder.Decode(&iface)
+	if err != nil {
+		return nil, err
+	}
+	return yamlpatch.NewNode(&iface).Container(), nil
+}
+
+func Test(c yamlpatch.Container, path, val string) (match bool, err error) {
+	defer func() {
+		if recover() != nil {
+			err = fmt.Errorf("No such path %s\n", path)
+			match = false
+		}
```

Comment:
Should add a comment here, since it's not obvious at a glance...
```suggestion
			// Implicit return
		}
```